### PR TITLE
📝 update readme documentation

### PR DESCRIPTION
**Pull request by stefan-prokop-cz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,143 +1,134 @@
-<div align="center">
-
-<img height="250" width="250" src="./assets/Badge_Trdelnik.png"/>
+[![Crates.io](https://img.shields.io/crates/v/trdelnik-cli?label=CLI)](https://crates.io/crates/trdelnik-cli)
```
Comment (tribuste) [inline README.md:1]: I'm thinking if it's not better to place it under Trdelnik logo

Comment (tribuste) [reply README.md:1]: I like how the Anchor repo looks like, and it is under Logo, but it just inspirational
https://github.com/project-serum/anchor

File: README.md [hunk 2]
```diff
@@ -1,143 +1,134 @@
...
+[![Crates.io](https://img.shields.io/crates/v/trdelnik-test?label=Test)](https://crates.io/crates/trdelnik-test)
+[![Crates.io](https://img.shields.io/crates/v/trdelnik-client?label=Client)](https://crates.io/crates/trdelnik-client)
+[![Crates.io](https://img.shields.io/crates/v/trdelnik-explorer?label=Explorer)](https://crates.io/crates/trdelnik-explorer)
+<br />
+[![lint](https://github.com/Ackee-Blockchain/trdelnik/actions/workflows/lint.yml/badge.svg)](https://github.com/Ackee-Blockchain/trdelnik/actions/workflows/lint.yml)
+[![test](https://github.com/Ackee-Blockchain/trdelnik/actions/workflows/test.yml/badge.svg)](https://github.com/Ackee-Blockchain/trdelnik/actions/workflows/test.yml)
```
Comment (tribuste) [inline README.md:7]: It looks like it's not positioned properly
Screenshot 2022-06-06 at 22 45 47

File: README.md (hunk 1 above)
Comment (stefan-prokop-cz) [reply README.md]: :facepalm: I think I was quite confused yesterday .. I thought logo is the name of some folder and you want from me to place the badges in this folder :smile: Moved .. sorry

File: README.md (hunk 2 above)
Comment (stefan-prokop-cz) [reply README.md]: I moved it under the logo

**Comment by stefan-prokop-cz** [comment]
@tribuste I added the possibility of skipping tests as a new chapter

**Pull request merged**